### ISSUE TITLE
Dep on byebug ~> 9.0 breaking for versions of ruby before 2.2

### ISSUE THREAD

**Issue by Mike-Dunton** [deivid-rodriguez/pry-byebug#123]
Get this error when trying to install pry-byebug 3.4.0

```
Installing byebug 9.1.0 with native extensions
--

Gem::InstallError: byebug requires Ruby version >= 2.2.0.
```

It looks like byebug 9.1.0 only supports Ruby 2.2.0. It was released today. https://rubygems.org/gems/byebug

**Comment by deivid-rodriguez** [comment]
Thanks! I'll release 3.4.3 as soon as possible fixing this!

**Comment by deivid-rodriguez** [comment]
Just to confirm. Does it still happen if you point to the master branch in your Gemfile (`gem "pry-byebug", git: "https://github.com/deivid-rodriguez/pry-byebug"`)? Does it get fixed if you point to #124 (`gem "pry-byebug", git: "https://github.com/deivid-rodriguez/pry-byebug", branch: "release/3.4.3"`)?

**Comment by Mike-Dunton** [comment]
@deivid-rodriguez Thanks for the quick response 

`gem "pry-byebug", git: "https://github.com/deivid-rodriguez/pry-byebug"` does not work
```
An error occurred while installing byebug (9.1.0), and Bundler cannot continue.
Make sure that `gem install byebug -v '9.1.0'` succeeds before bundling.
```

`gem "pry-byebug", git: "https://github.com/deivid-rodriguez/pry-byebug", branch: "release/3.4.3"` Does work.

**Comment by deivid-rodriguez** [comment]
Great! Expect a release in the next few hours!

**Comment by mvz** [comment]
IMHO, this should really be handled by Bundler and/or rubygems: Now I'm stuck on byebug 9.0 on Ruby 2.4 🙁.

**Comment by deivid-rodriguez** [comment]
Can you try out #126?

**Comment by mvz** [comment]
That works, thanks! 👍

**Comment by deivid-rodriguez** [comment]
I released `pry-byebug` 3.5.0, thanks for trying that out!
